classes/asserter/definition.php

```diff
@@ -2,8 +2,7 @@
 
 namespace mageekguy\atoum\asserter;
 
-use mageekguy\atoum
-;
+use mageekguy\atoum;
```

Comment:
Did you fix them by hand or with PHP CS Fixer? Why the `.php_cs` conf did not fix them?